pandas/core/indexing.py

```diff
@@ -1992,6 +1996,10 @@ def __setitem__(self, key, value):
 
         if not isinstance(key, tuple):
             key = _tuplify(self.ndim, key)
+
+        if isinstance(self.obj, ABCSeries) and len(key) > 1:
```

Comment:
is this only necessary with MultiIndex?

Reply:
I believe so, yes. When there is a regular index, key will be e.g. (1,). In the call to self.obj._get_value, the explosion *key then results in passing the correct value 1. If we were to apply the line key = [key] in this case, then (1,) would be incorrectly passed. Perhaps a comment to this effect should be added?